apps/rush-lib/src/logic/VersionManager.ts

```diff
@@ -14,19 +16,28 @@ import { RushConfigurationProject } from '../api/RushConfigurationProject';
 import { VersionPolicyConfiguration } from '../api/VersionPolicyConfiguration';
 import { PublishUtilities } from './PublishUtilities';
 import { ChangeManager } from './ChangeManager';
+import { TempProjectHelper } from './TempProjectHelper';
+import { PnpmShrinkwrapFile, IPnpmShrinkwrapDependencyYaml } from './pnpm/PnpmShrinkwrapFile';
 import { DependencySpecifier } from './DependencySpecifier';
+import { PurgeManager } from './PurgeManager';
+import { RushConstants } from './RushConstants';
+import { RushGlobalFolder } from '../api/RushGlobalFolder';
+import { RushInstallManager } from './installManager/RushInstallManager';
+import { IInstallManagerOptions } from './base/BaseInstallManager';
 
 export class VersionManager {
   private _rushConfiguration: RushConfiguration;
   private _userEmail: string;
   private _versionPolicyConfiguration: VersionPolicyConfiguration;
   private _updatedProjects: Map<string, IPackageJson>;
   private _changeFiles: Map<string, ChangeFile>;
+  private _globalFolder: RushGlobalFolder;
 
   public constructor(
     rushConfiguration: RushConfiguration,
     userEmail: string,
-    versionPolicyConfiguration?: VersionPolicyConfiguration
+    versionPolicyConfiguration: VersionPolicyConfiguration,
```

Comment:
This seems breaking. Any reason why it was made non-optional?

Reply:
Good point. I wanted to introduce a new, non-optional parameter. I am not sure if this is used publiclly and will break dependents. I looked at some internal projects and this contructor doesn't seem to be used anywhere outside of Rush. Thouhgts? CC: @iclanton @octogonz

Reply:
This is not part of the `rush-lib.api.md` API surface. It's okay to change it.

Reply:
Thanks Pete for confirming!